mets.md

```diff
@@ -86,24 +86,14 @@ with the type of manipulation (`BIN-KRAKEN`).
 `<mets:fileGrp USE>` | Type of use for OCR-D
 --                                          | --
 `<mets:fileGrp USE="OCR-D-IMG">`            | The unmanipulated source images
-`<mets:fileGrp USE="OCR-D-IMG-BIN">`        | Black-and-White images
-`<mets:fileGrp USE="OCR-D-IMG-CROP">`       | Cropped images
-`<mets:fileGrp USE="OCR-D-IMG-DESKEW">`     | Deskewed images
-`<mets:fileGrp USE="OCR-D-IMG-DESPECK">`    | Despeckled images
-`<mets:fileGrp USE="OCR-D-IMG-DEWARP">`     | Dewarped images
```

Comment:
These conventions still make sense for the file ID naming scheme within fileGrps, so they should **re-appear** there.

(See comment in [my first review](https://github.com/OCR-D/spec/pull/164#pullrequestreview-431636277).)

Reply:
Sorry, didn't see the other commits!

Reply:
How about introducing `PRE` as a workflow step? This is different than `IMG` (which is merely the original input image) and different than the old recommendation for separate image file groups. It would be used for PAGE and derived images alike (discerning the latter only with an ID suffix, see below).

```suggestion
`<mets:fileGrp USE="OCR-D-PRE-BIN">`        | Binarization preprocessing
`<mets:fileGrp USE="OCR-D-PRE-CROP">`       | Cropping preprocessing
`<mets:fileGrp USE="OCR-D-PRE-DESKEW">`     | Deskewing preprocessing
`<mets:fileGrp USE="OCR-D-PRE-DESPECK">`    | Despeckling preprocessing
`<mets:fileGrp USE="OCR-D-PRE-DEWARP">`     | Dewarping preprocessing
```

Reply:
@tboenig @EEngl52 what do you think?